test/start_server_test.js

```diff
@@ -61,4 +71,158 @@ suite('startServer', () => {
       .end(done);
   });
 
+  suite('h2', () => {
+    let _certFile;
+    let _keyFile;
+    let _nodeVersion;
+    let _serverOptions;
+    let _stubServer;
+
+    suiteSetup(() => {
+      _setupNodeVersion();
+      _setupServerOptions();
+      _setupStubServer();
+    });
+
+    suiteTeardown(() => {
+      _teardownStubServer();
+    });
+
+    test('rejects unsupported Node version (< 5) only', function() {
+      if (_nodeVersion < 5) {
+        return assert.isRejected(startServer(_serverOptions));
+      } else {
+        return assert.becomes(_startStubServer(_serverOptions), _stubServer);
+      }
+    });
+
+    suite('node5+', () => {
+      // test() requires old-style function to use `this.skip()`,
+      // where `this` is the current test instance, so don't use
+      // arrow-function for the test callback.
```

Comment:
you could just change the suite definition function depending on the node version:

```javascript
const suiteDef = _nodeVersion < 5 ? suite.skip : suite;
suiteDef('node5+', () => {
  // ... 
});
```